src/odemis/driver/lakeshore.py

```diff
@@ -69,33 +74,23 @@ class Lakeshore(model.HwComponent):
     def __init__(self, name, role, port, sensor_input='b', output_channel=2, **kwargs):
         """
         A driver for the Lakeshore 310 temperature controller.
-
         name: (str)
         role: (str)
         port: (str) port name. Can be a pattern, in which case all the ports
           fitting the pattern will be tried.
           Use /dev/fake for a simulator
```

Comment:
I think it would make sense to update this to include that this is either an IP address or the usb port, do you agree?

Reply:
Yes, that update would make it clearer.